### ISSUE TITLE
Does the code support image based prompting?

### ISSUE THREAD

**Issue by antoculus** [bytedance/MVDream#8]
Can I supply a one (or more) images to the model to learn a concept which can subsequently be used to generate a 3D model?
Thanks!

**Comment by pengwangucla** [comment]
yeah, we are working on image prompt tuning and obtained some models, it needs more optimization  potentially will provide one aroundn CVPR.
